jobs/JREGIONAL_CHECK_POST_OUTPUT

```diff
@@ -16,7 +31,9 @@
 #-----------------------------------------------------------------------
 #
 . $USHdir/source_util_funcs.sh
-source_config_for_task "" ${GLOBAL_VAR_DEFNS_FP}
+for sect in user nco workflow ; do
```

Comment:
This is an interesting dilemma. I understand what you're getting at here, but I don't think the quotes matter to bash. Everything is a string!

```
$ for sect in for; do echo $sect ; done
for
```
I also felt that it was a little weird to just have an "uncontained" list of words to iterate through and tried parentheses here like I was declaring an array in bash, but that doesn't work either:

```
$ for sect in ( foo bar baz ) ; do echo $sect ; done
bash: syntax error near unexpected token `('
```

I think this is just the appropriate syntax for a bash loop over strings. The quotes are unnecessary so I'd prefer to leave them out since they add no discernible advantage here.